suite-native/module-onboarding/src/screens/WelcomeScreen.tsx

```diff
@@ -91,15 +91,15 @@ export const WelcomeScreen = () => {
                 resizeMode="cover"
                 style={applyStyle(imageContainerStyle)}
             />
-            <Screen isScrollable={false} backgroundColor="transparent">
+            <Screen backgroundColor="transparent">
                 <LinearGradient
                     style={applyStyle(cardStyle)}
                     colors={[
                         utils.colors.gradientNeutralBottomFadeSurfaceElevation1Start,
                         utils.colors.gradientNeutralBottomFadeSurfaceElevation1End,
                     ]}
                 >
-                    <Box flex={0.7} />
+                    <Box flex={1.2} />
```

Comment:
Can you please add comment why this is needed? Seems obscure without explanation.